contrib/seeds/nodes_signet.txt

```diff
@@ -0,0 +1,10 @@
+# List of fixed seed nodes for signet
+
+165.232.44.123:38333
+106.157.214.199:38333
+31.14.40.19:38333
+134.209.26.10:38333
+178.128.221.177:38333
+103.16.128.63:38333
+153.126.143.201:38333
+192.241.163.142:38333
```

Comment:
Only:

```
    "address": "178.128.221.177",
    "address": "103.16.128.63",
    "address": "153.126.143.201",
    "address": "192.241.163.142",
```

seem to still be present?

Reply:
I think without a dns seeder or other software to track uptime, there will always be nodes in the list that are not reachable